packages/libs/eda/src/lib/map/analysis/mapTypes/plugins/BarMarkerMapType.tsx

```diff
@@ -356,7 +357,9 @@ function MapLayerComponent(props: MapTypeMapLayerProps) {
   });
 
   if (markerData.error && !markerData.isFetching)
-    return <MapFloatingErrorDiv error={markerData.error} />;
+    return isNoDataError(markerData.error) ? null : (
+      <MapFloatingErrorDiv error={markerData.error} />
+    );
```

Comment:
Above is where we ignore certain types of error and show no markers or spinners on the map (returning `null`)